csvcubed/tests/behaviour/steps/infojson2csvw.py

```diff
@@ -0,0 +1,103 @@
+import json
+from pathlib import Path
+
+from behave import when, then
+import subprocess
+from appdirs import AppDirs
+from typing import Tuple
+from csvcubed.utils.log import start_logging
+from csvcubeddevtools.behaviour.temporarydirectory import get_context_temp_dir_path
+
+
+@when('the csvcubed CLI is run with "{arguments}"')
+def step_impl(context, arguments: str):
+    command: str = f"csvcubed {arguments.strip()}"
+    (status_code, response) = run_command_in_temp_dir(context, command)
+    context.csvcubed_cli_result = (status_code, response)
+
+@when('the log location is determined by Appdirs')
+def step_impl(context):
+    dirs = AppDirs("csvcubedcli","csvcubed")
+    context.csvcubed_log_location = Path(dirs.user_log_dir)
```

Comment:
Shouldn't this be configured by the call to `start_logging()`? Why do we need to do it here?

Reply:
there needed to be a path to the log file file that could be stored in 'context' so that it could be pass between steps to check if the file exists and to later delete to log file when the test runs.